docs/manage/data/label.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Label Data"
+linkTitle: "Label Data"
+description: "Label image data on the DATA page in the Viam Cloud."
+weight: 39
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+images: [ "/manage/data/img/label-dog.gif" ]
+# SME: Tahiya Salam and Alexis Wei
+---
+
+To label data in a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+On the **Images** subtab, you can filter available images, using the **Filtering** menu and select the attributes that match where, how, and when the data was collected.
+
+{{< alert title="Important" color="note" >}}
+Datasets are views, not materialized.
+If the underlying data matching the filter changes because data is deleted or more data is added, the dataset will also change.
+
+Therefore the data you are viewing may change as you label and train on the dataset.
+{{< /alert >}}
+
+You can label you data with
+
+- [Bounding boxes](#bounding-boxes), which you can use to [train object detection models](/manage/ml/train-model/#train-a-model).
+- [Image tags](#image-tags), which you can use to [train object detection models](/manage/ml/train-model/#train-a-model).
```

Comment:
Image tags are use to train image classification models

Reply:
good catch!